packages/core/usage-data/core-usage-data-server-internal/src/core_usage_data_service.ts

```diff
@@ -43,7 +43,7 @@ import type { SavedObjectTypeRegistry } from '@kbn/core-saved-objects-base-serve
 import type { SavedObjectsServiceStart } from '@kbn/core-saved-objects-server';
 
 import { isConfigured } from './is_configured';
-import { coreUsageStatsType } from './core_usage_stats';
+import { coreUsageStatsType } from './saved_objects';
```

Comment:
I moved the stats SO type to a `saved_objects` subfolder, as that's our recommended approach